index.js

```diff
@@ -69,10 +71,11 @@ exports.handler = function(event, context) {
 					}
 				});
 			}, function(err) {
+				context.callbackWaitsForEmptyEventLoop = false;
 				if (err) {
-					context.fail(err);
+					callback(err, 'Failed result');
 				} else {
-					context.succeed();
+					callback(null, 'Success message');
```

Comment:
No. It was suggested by AWS. Basically the callback will report anything as success as long it doesn't pass an Error object. I will remove and update.